installer/ansible/roles/auth-installer/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2018 The OpenSDS Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- name: check port usage
+  wait_for:
+    port: "{{ item }}"
+    timeout: 5
+    state: stopped
+    msg: "Port {{ item }} is not free"
+  with_items:
+    - 80
+  become: yes
+
+- name: install keystone with the script
```

Comment:
If not required in multi-cloud, do not install keystone

Reply:
keystone required for mullti-cloud. even for API(postman)